### PR TITLE
Add Python WSGIServer fingerprint

### PR DESCRIPTION
**Pull request by mkienow-r7**
## Description
Adds fingerprint for Python's WSGIServer ([wsgiref.simple_server](https://docs.python.org/3/library/wsgiref.html#module-wsgiref.simple_server)), a simple WSGI HTTP server.

## Motivation and Context
Improved coverage

## How Has This Been Tested?
* `bundle exec ./bin/recog_verify xml/http_servers.xml`
* `rake tests`

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have updated the documentation accordingly (or changes are not required).
- [x] I have added tests to cover my changes (or new tests are not required).
- [x] All new and existing tests passed.

### CODE REVIEW

File: xml/http_servers.xml [hunk 1]
```diff
@@ -2623,6 +2623,17 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:encode:uvicorn:-"/>
   </fingerprint>
 
+  <fingerprint pattern="^WSGIServer/((?:\d+\.)*\d+)\s*C?Python/((?:\d+\.)*\d+)$">
```
Comment (cbarnard-r7) [inline xml/http_servers.xml:2626]: Matches `WSGIServer/0.2                          CPython/3.10.5` but I think that's fine.

**Review by cbarnard-r7 — APPROVED** [review]
LGTM

File: xml/http_servers.xml (hunk 1 above)
Comment (mkienow-r7) [reply xml/http_servers.xml:2626]: It should match `WSGIServer/0.2 CPython/3.10.5` as that's one of the examples. :)

Comment (cbarnard-r7) [reply xml/http_servers.xml:2626]: Oops, I copied the wrong example. I think it was a similar multiple dot situation.

**Pull request merged**